core/storage/audit/gae_models_test.py

```diff
@@ -57,6 +57,24 @@ def test_get_deletion_policy(self) -> None:
             audit_models.RoleQueryAuditModel.get_deletion_policy(),
             base_models.DELETION_POLICY.KEEP)
 
+    def test_get_export_policy(self) -> None:
+        sample_dict = base_models.BaseModel.get_export_policy()
```

Comment:
This just repeats the implementation of the original code, doesn't it?

Tests should typically be *independent* checks of the *interface* of the original method, and there should typically be no "logic" used in tests when trying to construct the expected value. In this case, perhaps you could check equality with the full expected export policy dict.

Reply:
Done!